tests/unit/neptune/new/test_stringify_unsupported.py

```diff
@@ -531,3 +533,20 @@ def test_integers_outside_32bits(self, run):
 
         with assert_no_warnings():
             run["data"] = data
+
+    def test_float_warnings(self, run):
+        with pytest.warns(NeptuneUnsupportedValue):
+            run["infinity"] = float("inf")
+            run["neg-infinity"] = float("-inf")
```

Comment:
What will happen here? Do we log an operation to assign value or like in series we're going to not log a unsupported value?

Reply:
Like in series:
- throw the warning to user, that is not a supported value
- skip the value